clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java

```diff
@@ -272,6 +272,7 @@ private boolean trySend(long now) {
 
     private void clientPoll(long timeout, long now) {
         client.poll(timeout, now);
+        now = time.milliseconds();
```

Comment:
It looks like `clearUnsentRequests` below doesn't actually use the passed timestamp, so could we remove it from the method's parameters? Then the extra call to time.milliseconds is unneeded.